README.md

```diff
@@ -11,7 +11,9 @@ CommandLineUtils
 [main-myget-badge]: https://img.shields.io/www.myget/natemcmaster/vpre/McMaster.Extensions.CommandLineUtils.svg?style=flat-square&label=myget
 
 
-This is a fork of [Microsoft.Extensions.CommandLineUtils](https://github.com/aspnet/Common), which is no longer under [active development](https://github.com/aspnet/Common/issues/257). This fork, on the other hand, will continue to make improvements, release updates and take contributions.
```

Comment:
Can you restore this sentence to the way it was? This is still true. The roadmap says I'm not going to build all the features myself and there doesn't appear to be a need for a bunch of work, but I am still open to making improvements and taking contributions. For example, someone just made a PR to add new API last week and it will be released at the end of the month.

If that wasn't clear, then the roadmap is the misleading part.

Reply:
OK, I'll restore it. I tend to agree. It is the road map that is misleading then. I think it could just be the introductory sentence:

> ...is the last version of this library I am planning to work on.

This makes it sound like you are not going to do _anything_ with it any more.

Reply:
Thanks for the feedback. I updated the roadmap. Does that make it clearer? Sorry for the confusion about what it all meant.

Reply:
The roadmap is much clearer now, thanks. 👍